lib/aca_entities/references/rating_area.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# TODO: Need to move right namespace
```

Comment:
rating area belongs under Locations namespace

Reply:
This is added as a placeholder, will be updating in upcoming releases.